docs/serverless-demo.md

```diff
@@ -53,7 +53,7 @@ subscription PlaybackStateSubscriberSubscription {
 }
 ```
 
-GraphOS Explorer also supports tabs. Open a new tab and try pausing the running subscription:
+GraphOS Explorer also supports tabs. Open a new tab and run the operation below to pause the running subscription:
```

Comment:
Clarify: new _Explorer_ tab. This should be the same _window_ tab as when we ran the subscription above, to better see that it did indeed get paused.